CHANGELOG.md

```diff
@@ -8,11 +8,18 @@ Information on how to properly use the Singularity cache with Cromwell is now
 provided in the [Cromwell Singularity documentation](
 https://cromwell.readthedocs.io/en/stable/tutorials/Containers/#singularity).
 
+### Google Library Upgrade [(#5565)](https://github.com/broadinstitute/cromwell/pull/5565)
+
+Previous versions of Cromwell use Google libraries that have been deprecated and will [stop working in August 2020](https://developers.googleblog.com/2018/03/discontinuing-support-for-json-rpc-and.html). This release adopts updated libraries to ensure continued smooth operation. No user action is required aside from upgrading Cromwell.   
+
```

Comment:
TOL so we may need to backport these upgrades into Cromwell 0.12 if the global endpoints really are shut down in August...

Reply:
If you check out `51_hotfix` and navigate to `BatchRequest.java:98` it specifically uses `https://www.googleapis.com/batch`.

In the `execute()` function at `BatchRequest.java:216` the value of `batchUrl` is used unmodified and we definitely don't call the `setBatchUrl()` mutator from Cromwell, sooooo.....

Reply:
"No user action is required aside from upgrading Cromwell. But yeah, you need to upgrade your Cromwell."